client/src/pages/hostSetup.tsx

```diff
@@ -0,0 +1,25 @@
+import styles from "../styles/hostSetup.module.css";
+import passwordStyle from "../styles/Textbox.module.css";
+import Textbox from "../components/Textbox";
+import Checkbox from "../components/Checkbox";
+import Button from "../components/Button";
+const hostSetup = () => {
+    return (
+        <div className={styles.main}>
+            <div className={styles.title}>Host a Room</div>
+            <div className={styles.grid}>
+                <Textbox placeholder="Room Name" />
```

Comment:
`<textarea>` is a multi-line input widget, which probably isn't what we want here